resources/main.css

```diff
@@ -1,3 +1,13 @@
+a {
+    color: white;
+    font-family: Roboto,sans-serif;
+}
+
+button {
```

Comment:
Yes, it's probably nice to have it by default. If you add text below a button for example, you'd like a margin between them too.
(If you want me to make this specific to this button though, tell me. :) )